### DISCUSSION TITLE
Enabling Meta Bind within tables of Live preview

### DISCUSSION THREAD

**Discussion by MTPGthb** [mProjectsCode/obsidian-meta-bind-plugin#271] [Q&A] [answered]
I currently use obsidian in live preview and would love to use it within a table but when I try to enter a value or select an option from the drop down option, it goes back to the source code. It is able to work within inline features but not within a table.

Is there a way to enable it the plugin to work within a table? Also note, I have tried using "meta data menu" and which was working for a long time, but it began to bug out with different errors.

thank you!

**Comment by mProjectsCode** [comment, marked as answer]
I tried to fix that issue, but it seems like a fix from the obsidian side is needed for that.
